From 6bcea60b6ca35469856837b2fd10104cde3c6b22 Mon Sep 17 00:00:00 2001
From: Suresh Gupta <suresh.gupta@xilinx.com>
Date: Wed, 27 Feb 2019 06:21:24 -0800
Subject: [PATCH 0510/1852] documentation: dt: multi-scaler: Add clock
 framework support

commit 104ed743ffc84c7d8fc1c39e0c0720f6918b8691 from
https://github.com/Xilinx/linux-xlnx.git

This patch adds the clock property to the documentation.

By introducing these changes, the backward compatibility is broken
between DT and driver. So, it is mandatory to take new DT with
latest driver.

Signed-off-by: Suresh Gupta <suresh.gupta@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt  | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt
index 6e1b55f12d8a..3aea1f36a6ce 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-multi-scaler.txt
@@ -3,6 +3,7 @@ Xilinx mem2mem Multi Video Scaler (XM2MSC)
 
 Required propertie(s):
 - compatible		: Should be "xlnx,v-multi-scaler-v1.0"
+- clocks		: Input clock specifier. Refer to common clk bindings.
 - interrupt-parent	: Interrupt controller the interrupt is routed through
 - interrupts		: Should contain MultiScaler interrupt
 - reset-gpios		: Should contain GPIO reset phandle
@@ -76,7 +77,8 @@ Y_UV10_420	xv15		Semi-planar YUV 4:2:0 with 10 bits per component.
 
 Example
 
-v_multi_scaler_0: v_multi_scaler@a0000000 {
+v_multi_scaler_0: v_multi_scaler@a0000000 {\
+	clocks = <&clk 71>;
 	compatible = "xlnx,v-multi-scaler-v1.0";
 	interrupt-names = "interrupt";
 	interrupt-parent = <&gic>;
-- 
2.31.1

